src/components/movie-description/movie-description.tsx

```diff
@@ -0,0 +1,38 @@
+import { Logo } from 'components/logo/logo';
+import { useMovieContext } from 'contexts/movie-context';
+import { FC } from 'react';
+import { ReactComponent as Search } from 'assets/search.svg';
+import styles from './movie-description.module.scss';
+
+export const MovieDescription: FC = () => {
+  const { selectedMovie, setSelectedMovie } = useMovieContext();
+
+  return (
+    <div className={styles.container}>
+      <div className={styles.headerWpapper}>
+        <Logo />
+        <Search
+          className={styles.search}
+          onClick={() => setSelectedMovie(undefined)}
+        />
+      </div>
+      <div className={styles.movieWrapper}>
+        <img src={selectedMovie?.img} alt={selectedMovie?.title} />
```

Comment:
I am not really sure that we need to mark `selectedMovie` with optional chaining operator here because we don't need to render this component at all if there is no selected movie from context. Are you sure that we don't render this component if selected movie is `undefined`? If yes, let's remove optional chaining operator. If not, please, update the logic to render it only for existing item in context